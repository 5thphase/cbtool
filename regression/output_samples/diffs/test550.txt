 status: Trying to establish network connectivity to vm_XXX (cloud-assigned uuid xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx), on IP address xxx.xxx.xxx.xxx ...
 status: Waiting for vm_XXX (cloud-assigned uuid xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx) to start...
 status: Bypassing the bootstrapping and the sending of a copy of the code tree to vm_59 (xxx.xxx.xxx.xxx )...
 status: Trying to establish network connectivity to vm_XXX (cloud-assigned uuid xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx), on IP address xxx.xxx.xxx.xxx ...
 status: Bypassing the bootstrapping and the sending of a copy of the code tree to vm_58 (xxx.xxx.xxx.xxx )...
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1406645236 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1406645277 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1406645295 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1406645384 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1406645390 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1406645395 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1406645402 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
