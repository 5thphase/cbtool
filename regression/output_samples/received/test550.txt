 status: Waiting for vm_59 (cloud-assigned uuid 429653BC-EE62-5433-A748-BACC95B313C7) to start...
 status: Waiting for vm_58 (cloud-assigned uuid D59CF29E-5E60-51AE-B57E-5DD009E00B71) to start...
 status: Waiting for vm_60 (cloud-assigned uuid 752334DA-EB43-587B-AF78-6E86FA186A38) to start...
 status: Trying to establish network connectivity to vm_58 (cloud-assigned uuid D59CF29E-5E60-51AE-B57E-5DD009E00B71), on IP address 246.25.217.83...
 status: Trying to establish network connectivity to vm_60 (cloud-assigned uuid 752334DA-EB43-587B-AF78-6E86FA186A38), on IP address 63.155.133.191...
 status: Trying to establish network connectivity to vm_59 (cloud-assigned uuid 429653BC-EE62-5433-A748-BACC95B313C7), on IP address 153.115.53.90...
 status: Bypassing the sending of a copy of the code tree to vm_58 (246.25.217.83)...
 status: Bypassing the sending of a copy of the code tree to vm_59 (153.115.53.90)...
 status: Bypassing the sending of a copy of the code tree to vm_60 (63.155.133.191)...
 status: Bypassing generic VM post_boot configuration on all VMs belonging to ai_11...
 status: Bypassing application-specific configuration on all VMs beloging to ai_11...
 status: QEMU Scraper will NOT be automatically started during the deployment of ai_11...
AI object C976E903-B14B-5E72-A602-27D5335DF3C8 (named "ai_11") sucessfully attached to this experiment. It is ssh-accessible at the IP address 246.25.217.83 (D59CF29E-5E60-51AE-B57E-5DD009E00B71.simcloud.com).
The following ATTACHED AIs are in this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_3      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |247.245.127.176 |1393125657     |none                                    |BC986610-A93E-54C5-8C32-2EB7F3B0E6D9
|ai_4      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |18.182.141.190  |1393125697     |none                                    |1BAA574E-A47D-56E3-B6C3-FC73D38D0076
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |161.245.121.152 |1393125716     |none                                    |AAB7E4FD-FC5A-5D57-8ADA-8B057D69E919
|ai_8      |daytrader      |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |68.19.244.164   |1393125802     |none                                    |381E1F4E-354B-55EA-BE05-FAD8798A6EEB
|ai_9      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |187.168.171.82  |1393125808     |none                                    |05395B40-46AC-5B19-AB15-61DCC0F10DF2
|ai_10     |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |162.12.73.198   |1393125813     |none                                    |D1E3BFB5-BBCD-5A81-8C5E-FB2B3D6395C5
|ai_11     |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |246.25.217.83   |1393125819     |none                                    |C976E903-B14B-5E72-A602-27D5335DF3C8

The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_33         |db2                 |platinum64  |78.49.41.226    |simhostc1                  |SUT                 |ai_3    |none       |7ABA32C5-5938-55AE-82B8-6AE0189F45B8
|vm_34         |client_daytrader    |nano32      |247.245.127.176 |simhostd1                  |LG                  |ai_3    |none       |49491B49-B3CA-528E-B109-F65ADCA233F0
|vm_35         |was                 |iron32      |205.146.94.73   |simhosta1                  |SUT                 |ai_3    |none       |C2914876-0428-55D1-8E7F-8231B2F9D03F
|vm_37         |was                 |iron32      |50.178.133.175  |simhostb1                  |SUT                 |ai_4    |none       |5D0DCF79-67B5-584D-AAEC-FBD053271614
|vm_36         |client_daytrader    |nano32      |18.182.141.190  |simhostd1                  |LG                  |ai_4    |none       |60415AAF-382B-53C3-A59D-43C62EC5B302
|vm_38         |db2                 |platinum64  |237.82.26.75    |simhostb1                  |SUT                 |ai_4    |none       |9D9B8BB3-417D-5857-9D2D-DB907FB2B3D7
|vm_39         |netserver           |iron32      |241.38.202.26   |simhostb1                  |SUT                 |ai_5    |none       |6C86699A-7AEB-5D5B-8899-1B9B28C516E1
|vm_40         |netclient           |platinum64  |161.245.121.152 |simhostb1                  |SUT                 |ai_5    |none       |1A3DB01D-0407-59DA-8670-66C312A36BF3
|vm_47         |was                 |iron32      |158.9.36.163    |simhostc1                  |SUT                 |ai_8    |none       |E2FD591D-D1D0-5F54-A7E9-56D260EA4687
|vm_48         |lb                  |iron32      |117.182.26.148  |simhostc1                  |SUT                 |ai_8    |none       |D99E0ACB-6F3E-5037-9858-12B5521034BF
|vm_49         |client_daytrader    |nano32      |68.19.244.164   |simhostd1                  |LG                  |ai_8    |none       |73BAC24D-3E20-5062-BCEC-ECCCAFAD4D44
|vm_50         |db2                 |gold32      |12.204.192.197  |simhosta1                  |SUT                 |ai_8    |none       |AB0D0B51-2A57-5F8B-8DE3-019737AD32A8
|vm_51         |was                 |iron32      |234.188.234.153 |simhostc1                  |SUT                 |ai_8    |none       |4FEC4843-ED3D-5DB6-9431-E80FCDF10D2B
|vm_52         |client_daytrader    |nano32      |187.168.171.82  |simhostd1                  |LG                  |ai_9    |none       |12606459-0CBB-50D9-9C57-6BBA7C2269F1
|vm_53         |was                 |iron32      |178.248.210.20  |simhostb1                  |SUT                 |ai_9    |none       |2AD4B707-1E0F-525B-852C-CFCE28C92268
|vm_54         |db2                 |platinum64  |64.106.183.79   |simhostb1                  |SUT                 |ai_9    |none       |E72B5FD6-61CC-5F09-A860-A5DA98846F0A
|vm_57         |db2                 |platinum64  |247.247.29.22   |simhostc1                  |SUT                 |ai_10   |none       |515240C6-624E-5841-A870-65061F9F9C83
|vm_56         |client_daytrader    |nano32      |162.12.73.198   |simhostd1                  |LG                  |ai_10   |none       |0F118F90-020C-5C44-BFF5-4B38CCA0275A
|vm_55         |was                 |iron32      |156.67.9.205    |simhostb1                  |SUT                 |ai_10   |none       |ECD22DCB-C94C-5A4D-B731-83F02A82F5B0
|vm_59         |db2                 |platinum64  |153.115.53.90   |simhostc1                  |SUT                 |ai_11   |none       |4AFE9534-1D8C-58B4-B30D-D131F7DAB2E2
|vm_58         |client_daytrader    |nano32      |246.25.217.83   |simhostd1                  |LG                  |ai_11   |none       |FED6BC21-B442-5CDD-8CC1-19A303B794D3
|vm_60         |was                 |iron32      |63.155.133.191  |simhostc1                  |SUT                 |ai_11   |none       |5B4BC185-0DA2-56AA-854E-6830453562FA

The AI object ai_11, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (AI object key)               |value                                
|load_balancer                           |False                                
|sut                                     |1_x_client_daytrader->1_x_was->1_x_db2

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |3.79M                         
|Redis Version                                        |2.6.16                        
|Uptime (in seconds)                                  |2597502                       
|Total Connections Received                           |10003                         
|Total Commands Processed                             |5496551                       
|Number of Keys                                       |611                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.6                         
|Storage Size                                         |1642496                       
|Data Size                                            |651968                        
|Index Size                                           |457856                        
|Average Object Size                                  |530.918566775                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |22                            
|AIs                                                  |7                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |22                            
|VMs ARRIVED                                          |57                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |35                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|AI RESERVATIONS                                      |7                             
|AIs ARRIVED                                          |11                            
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |4                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |198                           

 ---------------- VMC-WIDE COUNTERS ----------------
|3B3CF412-FEA5-5D5E-8F68-B06301E1C33B (simzone_c) VM RESERVATIONS|7                             
|7A3E8461-A11D-5301-9F86-0773F1AEDD4A (simzone_a) VM RESERVATIONS|2                             
|C9ED5C50-B4F6-567D-9637-7B0234F29CA2 (simzone_b) VM RESERVATIONS|7                             
|6FDFFC32-C029-57B2-A4AA-0A74235A02A0 (simzone_d) VM RESERVATIONS|6                             

