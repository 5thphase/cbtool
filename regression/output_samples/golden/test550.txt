 status: Waiting for vm_59 (cloud-assigned uuid 39623EDE-2180-5D66-82D6-6122F3B6058F) to start...
 status: Waiting for vm_60 (cloud-assigned uuid 6A1AE68E-39AD-5EEE-8F04-7C207BD12C51) to start...
 status: Waiting for vm_58 (cloud-assigned uuid F5778787-03F9-5B27-8C65-D824F7491A58) to start...
 status: Trying to establish network connectivity to vm_59 (cloud-assigned uuid 39623EDE-2180-5D66-82D6-6122F3B6058F), on IP address 245.212.146.66...
 status: Trying to establish network connectivity to vm_58 (cloud-assigned uuid F5778787-03F9-5B27-8C65-D824F7491A58), on IP address 210.138.249.161...
 status: Trying to establish network connectivity to vm_60 (cloud-assigned uuid 6A1AE68E-39AD-5EEE-8F04-7C207BD12C51), on IP address 111.189.220.23...
 status: Bypassing the sending of a copy of the code tree to vm_59 (245.212.146.66)...
 status: Bypassing the sending of a copy of the code tree to vm_60 (111.189.220.23)...
 status: Bypassing the sending of a copy of the code tree to vm_58 (210.138.249.161)...
 status: Bypassing generic VM post_boot configuration on all VMs belonging to ai_11...
 status: Bypassing application-specific configuration on all VMs beloging to ai_11...
 status: QEMU Scraper will NOT be automatically started during the deployment of ai_11...
AI object 24A22ADF-E4E2-5A57-B3C9-715E0E55A9A9 (named "ai_11") sucessfully attached to this experiment. It is ssh-accessible at the IP address 245.212.146.66 (39623EDE-2180-5D66-82D6-6122F3B6058F.simcloud.com).
The following ATTACHED AIs are in this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_3      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |77.136.100.117  |1381326860     |none                                    |870FADCB-8EFA-51B5-A2CC-C367230522F7
|ai_4      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |231.86.90.94    |1381326903     |none                                    |37DC0C46-B289-52BE-B23A-838CCC59A153
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |128.1.118.199   |1381326922     |none                                    |817F3716-E289-5E74-9CFA-439D68B796FF
|ai_8      |daytrader      |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |40.226.122.43   |1381327012     |none                                    |BC3E5204-4D7F-55E4-89A9-76B4E6ADE788
|ai_9      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |76.45.10.114    |1381327017     |none                                    |7EC593E7-D1D6-5AE6-95C1-26A50C79CA8D
|ai_10     |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |129.104.39.58   |1381327023     |none                                    |8D553CE9-062A-5B54-A4FE-1C25F96EF9F0
|ai_11     |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |245.212.146.66  |1381327029     |none                                    |24A22ADF-E4E2-5A57-B3C9-715E0E55A9A9

The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_33         |client_daytrader    |nano32      |77.136.100.117  |simhostd1                  |LG                  |ai_3    |none       |28C3D503-AF07-539B-8058-BF5F744612EF
|vm_34         |db2                 |platinum64  |63.78.136.245   |simhosta1                  |SUT                 |ai_3    |none       |FCDF7F97-76C0-59AA-98AD-54F86FB54941
|vm_35         |was                 |iron32      |9.47.241.191    |simhosta1                  |SUT                 |ai_3    |none       |8E53C5F9-7B69-5C0D-A576-F89F5E1C4661
|vm_36         |client_daytrader    |nano32      |231.86.90.94    |simhostd1                  |LG                  |ai_4    |none       |D74FF3C7-0748-5DF4-95AA-3C5CA991145F
|vm_37         |was                 |iron32      |28.52.95.113    |simhostc1                  |SUT                 |ai_4    |none       |3DCEBD35-7918-5ECA-B5BE-754974C9551D
|vm_38         |db2                 |platinum64  |237.245.160.7   |simhosta1                  |SUT                 |ai_4    |none       |F7D8E555-DA9B-5B14-ABF6-21B93BC4BD78
|vm_39         |netserver           |iron32      |176.49.35.238   |simhostc1                  |SUT                 |ai_5    |none       |B429FD85-77F3-5CDA-BDA3-BE6BAE29F359
|vm_40         |netclient           |platinum64  |128.1.118.199   |simhosta1                  |SUT                 |ai_5    |none       |44CD170B-2DBF-5057-A049-F0158C7B2C65
|vm_48         |was                 |iron32      |215.98.31.84    |simhosta1                  |SUT                 |ai_8    |none       |B2243313-7D32-5191-B701-8CFCBD90929B
|vm_47         |was                 |iron32      |132.111.246.150 |simhosta1                  |SUT                 |ai_8    |none       |AA4D7546-6AE4-5C37-BF8E-8F738DA6DDA2
|vm_50         |lb                  |iron32      |42.85.220.153   |simhosta1                  |SUT                 |ai_8    |none       |4F969A74-1BA5-5159-82A5-3F3A920FAE16
|vm_49         |client_daytrader    |nano32      |40.226.122.43   |simhostd1                  |LG                  |ai_8    |none       |A9853882-9AED-5574-B00F-47E9E6901979
|vm_51         |db2                 |gold32      |28.120.188.56   |simhostb1                  |SUT                 |ai_8    |none       |FD8C8D85-297F-5F1B-BECF-097657CB1F3F
|vm_52         |client_daytrader    |nano32      |76.45.10.114    |simhostd1                  |LG                  |ai_9    |none       |7912B8BF-2F10-5009-B993-0975B5F493CB
|vm_53         |was                 |iron32      |44.65.231.70    |simhostc1                  |SUT                 |ai_9    |none       |E21DF9A9-46A8-507D-8FA2-FCABA0A92EB9
|vm_54         |db2                 |platinum64  |136.72.89.49    |simhostb1                  |SUT                 |ai_9    |none       |9066E1E4-6EF0-524C-BAD8-2AC982900577
|vm_55         |client_daytrader    |nano32      |129.104.39.58   |simhostd1                  |LG                  |ai_10   |none       |D6F5AFA3-C392-5608-8A99-26EBC9039489
|vm_56         |db2                 |platinum64  |44.90.22.168    |simhosta1                  |SUT                 |ai_10   |none       |DEF06AE2-0B9F-5D16-BF96-5BA332CBCB94
|vm_57         |was                 |iron32      |175.16.7.112    |simhosta1                  |SUT                 |ai_10   |none       |52D9A1F2-1962-5DD5-A39B-89F1D158C570
|vm_58         |was                 |iron32      |210.138.249.161 |simhosta1                  |SUT                 |ai_11   |none       |EF610B97-06D3-5306-8CFD-BC4CE6BAC187
|vm_60         |db2                 |platinum64  |111.189.220.23  |simhostb1                  |SUT                 |ai_11   |none       |632229C9-E95F-52E2-8C6E-C07D7F58A933
|vm_59         |client_daytrader    |nano32      |245.212.146.66  |simhostd1                  |LG                  |ai_11   |none       |4610480A-9A66-50D3-A669-06BD44FDAD25

The AI object ai_11, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (AI object key)               |value                                
|load_balancer                           |False                                
|sut                                     |1_x_client_daytrader->1_x_was->1_x_db2

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |4.44M                         
|Redis Version                                        |2.5.12                        
|Uptime (in seconds)                                  |2417637                       
|Total Connections Received                           |4591                          
|Total Commands Processed                             |3973099                       
|Number of Keys                                       |611                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.5                         
|Storage Size                                         |5448777728.0                  
|Data Size                                            |4962586376.0                  
|Index Size                                           |133824768                     
|Average Object Size                                  |8100.84602539                 
|Collections                                          |40                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |22                            
|AIs                                                  |7                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |22                            
|VMs ARRIVED                                          |57                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |35                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|AI RESERVATIONS                                      |7                             
|AIs ARRIVED                                          |11                            
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |4                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |198                           

 ---------------- VMC-WIDE COUNTERS ----------------
|CEA17C37-820A-507F-9356-90956267A568 (simzone_a) VM RESERVATIONS|10                            
|966D86CB-90BB-553D-A06A-0FFA0818D568 (simzone_d) VM RESERVATIONS|6                             
|C41BC369-F411-5382-B95F-70C31BD20E23 (simzone_c) VM RESERVATIONS|3                             
|2FF9728E-95B3-5EE9-A0A4-61875B507FA8 (simzone_b) VM RESERVATIONS|3                             

